modular_skyrat/modules/customization/modules/mob/living/carbon/human/species/roundstartslime.dm

```diff
@@ -1,5 +1,6 @@
 /datum/species/jelly
 	species_traits = list(MUTCOLORS,EYECOLOR,NOBLOOD,HAIR,FACEHAIR)
+	inherent_traits = list(TRAIT_CAN_USE_FLIGHT_POTION)
```

Comment:
I also feel like this could cause the same problem as for lizards, ensure they have the same `inherent_traits` list as everyone else instead. That means, just check if any instance of `/datum/species/jelly` is defined anywhere else so we can just either change that there or modularize it all.